airflow_aws_executors/batch_executor.py

```diff
@@ -101,9 +101,10 @@ def sync(self):
 
     def _describe_tasks(self, job_ids) -> List[BatchJob]:
         all_jobs = []
-        max_batch_size = self.__class__.DESCRIBE_JOBS_BATCH_SIZE
-        for i in range((len(job_ids) // max_batch_size) + 1):
-            batched_job_ids = job_ids[i * max_batch_size: (i + 1) * max_batch_size]
+        for i in range(0, len(job_ids), self.__class__.DESCRIBE_JOBS_BATCH_SIZE):
+            batched_job_ids = job_ids[i: i + self.__class__.DESCRIBE_JOBS_BATCH_SIZE]
+            if not batched_job_ids:
```

Comment:
this is now overly defensive (this should never be none) but doesn't harm to keep this in...


```python
>>> for batch_size in range(1, 101):
    for job_size in range(0, 1000):
        job_ids = range(0, job_size)
        for i in range(0, len(job_ids), batch_size):
            batched_job_ids = job_ids[i: i + batch_size]
            assert len(batched_job_ids) > 0
... 
>>> 
```